model-optimizer/extensions/front/kaldi/replace_lstm_nonlinearity.py

```diff
@@ -27,12 +27,26 @@ def run_before(self):
         return [FullyConnectedDecomposer]
 
     def replace_op(self, graph: Graph, node: Node):
-        # split input to (i_part, f_part, c_part, o_part, ct_1)
         node_name = node.soft_get('name', node.id)
+        # check if we have dropout
+        input_port = node.in_port(0)
+        if node['use_dropout']:
+            split_dropout = AttributedVariadicSplit(graph,
+                                                    {'name': node_name + '/split_dropout',
+                                                     'size_splits': np.array([-1, 1, 1, 1],),
+                                                     'axis': np.int64(1)}).create_node()
+            input_port.get_connection().set_destination(split_dropout.in_port(0))
+            input_port = split_dropout.out_port(0)
+
+        # split input to (i_part, f_part, c_part, o_part, ct_1)
         split_node = create_op_with_const_inputs(graph, Split, {1: np.int64(1)},
                                                  {'name': node_name + '/split_lstm_input',
                                                   'num_splits': 5})
-        node.in_port(0).get_connection().set_destination(split_node.in_port(0))
+        input_port.get_connection().set_destination(split_node.in_port(0))
+
+        i_part = split_node.out_port(0)
+        f_part = split_node.out_port(1)
+        o_part = split_node.out_port(3)
 
```

Comment:
please create `c_part = split_dropout.out_port(2)` this is used below

Reply:
please create `ct_1 = split_dropout.out_port(4)`. And use all ports below for better code readability.